back/db/migrate/20230623085057_add_votes_to_baskets_ideas.rb

```diff
@@ -5,5 +5,8 @@ def change
     add_column :baskets_ideas, :votes, :integer, null: false, default: 1
     execute 'UPDATE ideas SET budget = 1 WHERE budget < 1'
     execute 'UPDATE baskets_ideas SET votes = ideas.budget FROM ideas WHERE ideas.id = baskets_ideas.idea_id'
+
+    remove_index :baskets_ideas, name: :index_baskets_ideas_on_basket_id
+    add_index :baskets_ideas, %i[basket_id idea_id], unique: true
```

Comment:
Are we sure there are no basket_ideas at the moment that are duplicates and would cause this to fail?

Reply:
Yes, I looked at the counts on Metabase and didn't see any duplicates. But if you want I can add a query to remove duplicates just to be sure (this takes some time to test etc.).